### ISSUE TITLE
addIterativeLSI changing seed does not change SVD result

### ISSUE THREAD

**Issue by bnprks** [GreenleafLab/ArchR#326]
`addIterativeLSI` takes a `seed` parameter, but the SVD results do not change even if you pass a different `seed` value. Note that if cell sampling is used changing the seed will still change the output, but without cell sampling changing the seed value has no effect.

This is a bit of a twofer documentation / minor bug. 
On the documentation side: Section 4.2 of the manual states 
> It is important to note that LSI is not deterministic. This means that even if you run LSI in exactly the same way with exactly the same parameters, you will not get exactly the same results.

However, the existence of a `seed` argument and my own empirical experiments suggest this is not true. If there are certain cases where setting the same seed doesn't guaranty reproducibility then a clarification rather than a blanket statement might be better.

On the code bug side:
I believe the origin of this behavior is that the internal `.computeLSI` function has its own `seed` argument which is by default set to 1. Since `.LSIPartialMatrix` doesn't provide a seed value when calling `.computeLSI`, `.computeLSI` returns exactly the same results regardless of the seed passed to `addIterativeLSI`

The quick version of code I used to determine this:
```R
proj_10x <- readRDS("04_data/10x_PBMCs/ArchR_Project/Save-ArchR-Project.rds")

set.seed(105125)
keeper_cells <- sample(getCellNames(proj_10x), 1000)

subset_1k_v1 <- subsetCells(proj_10x, keeper_cells) %>%
  addIterativeLSI(name="LSI_Reproducibility_Test_1", saveIterations=FALSE, outDir="/tmp", seed=1)
subset_1k_v2 <- subsetCells(proj_10x, keeper_cells) %>%
  addIterativeLSI(name="LSI_Reproducibility_Test_1", saveIterations=FALSE, outDir="/tmp", seed=12)

digest::digest(subset_1k_v1) == digest::digest(subset_1k_v2) # TRUE
all(subset_1k_v1@reducedDims$LSI_Reproducibility_Test_1$matSVD == 
    subset_1k_v2@reducedDims$LSI_Reproducibility_Test_1$matSVD)
```
I've also confirmed that calling `.computeLSI` with different seed values does indeed produce different results

**Comment by rcorces** [comment]
Thanks @bnprks 

With respect to the bug, I believe your interpretation is correct. I've noticed similar issues in other functions (like #273) but had not caught this one. 

With respect to the documentation:
> my own empirical experiments suggest this is not true
Hmm. That was not my original experience. I did test each step of the process and came to a different conclusion. But in re-testing this now, I do see the same thing that you are seeing. I'm not sure exactly what lead me to believe it was nondeterministic but I will look into this further and correct the documentation.
